### DISCUSSION TITLE
Allocate full time developers and designers to cardano.org

### DISCUSSION THREAD

**Discussion by RayTracedObserver** [cardano-foundation/cardano-org#112] [Ideas]
The cardano.org website's development and improvement seems extremely slow and this may be a hindrance to the development of the cardano ecosystem as the website serves, or should serve, as a window into the cardano protocol and ecosystem. I believe that the cardano foundation should hire or allocate some full time developers and designers to the task of improving the website. I am not connected in any way to the foundation so I don't know how things currently function in regards to whom is tasked to work on the website but I am under the impression that there currently are very few contributors that are committed to this github with @katomm (who seems to be part of the foundation ?) being the main one.

A project as big as cardano should have a better designed website and be more frequently updated (don't get me wrong, the work that has already been done is good, but for sure a refresh of some graphical/visual elements would be welcome, and many improvements are still needed to both content and design). It is good to have open sourced the website, but for now I don't think its content should entirely rely on the contributions of benevolent people who are not part of cardano's founding entities as there aren't enough incentives to participate. A team allocated to this task and paid by the cardano foundation would surely be of great help. At least for a little while.

**Comment by RayTracedObserver** [comment]
@katomm what is your opinion on this ?

**Comment by katomm** [comment]
My opinion on this is that cardano.org website should be managed by a collective and not a single entity. Don't be fooled by my activity here in the issues and discussions, there are multiple people on this. The Contributors are slowly but surely increasing.  

See https://github.com/cardano-foundation/cardano-org/releases for what is ongoing. https://developers.cardano.org started exactly in the same way, this will take months from now still. Thank you for being one of the pioneers @RayTracedObserver
